g2p_registry_individual/models/individual.py

```diff
@@ -60,3 +58,13 @@ def _check_birthdate(self):
             if record.birthdate and record.birthdate > date.today():
                 error_message = "Birth date must be on or before the current date."
                 raise ValidationError(error_message)
+
+    def get_gender_selection(self):
+        gender_options = self.env["ir.config_parameter"].get_param(
+            "g2p_registry.gender_config"
+        )
+        if gender_options:
+            get_value = gender_options.split(",")
+            return [(opt, opt.capitalize()) for opt in get_value]
```

Comment:
This could be an issue. Why not take in a map/json in the config parameter, saying {"male":"Male", "female":"Female"}? Or the selection array itself like [("male","Male"),("female","Female")] ?

Reply:
@shibu-narayanan need your inputs.

Reply:
The design thought is to configure Gender list in the settings as coma separated values to bring in uniformity while getting added through various means (UI/API).